village/src/App.js

```diff
@@ -11,6 +13,14 @@ class App extends Component {
       smurfs: [],
     };
   }
+  componentDidMount(){
+    axios
+    .get("http://localhost:3333/smurfs");
+    .then(response =>{
+      this.setState({smurfsData: response.data});
```

Comment:
when you're setting your state, you want this to equal what's inside your state (rather than smurfsData, it should be smurfs)